test/test_common.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
+
+# Unit tests for the common module
+class CommonTest < MiniTest::Test
```

Comment:
thanks. It should be tested here since it tests code that is here and we want the test to fail if that code breaks.
this should be dropped as part of the group api patch (that would need to change as it currently relays on pluralize)